src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -98,12 +98,28 @@ public int Main(string[] args)
                 throw;
             }
 #else
+            catch (AggregateException aex)
+            {
+                foreach (var exception in aex.InnerExceptions)
+                {
+                    DumpException(exception);
+                }
+                return 1;
+            }
             catch (Exception ex)
             {
-                AnsiConsole.GetError(useConsoleColor: _runtimeEnv.OperatingSystem == "Windows").WriteLine($"Error: {ex.Message}".Red().Bold());
+                DumpException(ex);
                 return 1;
             }
 #endif
         }
+
+        private void DumpException(Exception ex)
+        {
+            AnsiConsole
+                .GetError(useConsoleColor: _runtimeEnv.OperatingSystem == "Windows")
+                .WriteLine($"Error: {ex.Message}".Red().Bold());
```

Comment:
For example, when `DNX_TRACE` is off, we want to see:

```
Error: Something bad happened
```

But with `DNX_TRACE`:

```
Error: Something bad happened
[Error] Full Exception: System.SomethingBadHappenedException: Something bad happend
 at blah.blah.call.stack
 at blah.blah.call.stack
```